server.js

```diff
@@ -20,6 +23,35 @@ app.get('/ping', (req, res) => {
   res.send('This is the About page');
 })
 
+
+// Connection URI
+const uri = 'mongodb+srv://NAYANKUMARRAJ:nkr2580@my-first-cluster.hz1puza.mongodb.net/?retryWrites=true&w=majority';

```

Comment:
Avoid hardcoding sensitive information like the MongoDB URI directly in the code. This should be stored in environment variables for security reasons. [important]